### DISCUSSION TITLE
day 30-2 30-3 문제를 못풀겠습니다 ㅠㅠ..

### DISCUSSION THREAD

**Discussion by Mang-Joo** [I-JUNNYLAND-I/algorithm#157] [Q&A]
문제 어떻게 풀어야 할 지 감이 안오네요

**Comment by Junnyjun** [comment]
30 -2 )
숫자 두개인 경우 부터 확인해 보도록 합니다.
4,3 인경우
4,4 
4,5 
-> 총 2번의 작업이 되어야 합니다.

숫자 세개인 경우
4,2,1
`4,2`,1 중 태그를 작업합니다 -> 4,5,1   (2->5 3만큼)
4,`5,1` 중 태그를 작업합니다. -> 4,5,6  (1->6 5만큼)
총 8입니다.


숫자 네개인 경우 [ 1,4,1,2]
1 `4 2` 2
1 `4 3` 2
1 `4 4` 2 
1 `4 5` 2
Arr[1] - Arr[2] +1 = 4
1 4 `5 3`
1 4 `5 4`
1 4 `5 5`
1 4 `5 6`
Arr[2]`5로바뀐상태` - Arr[3] +1 = 4

문제 풀기 전 `점화식`을 댓글로 달아주세요!

**Reply by Junnyjun** [reply]
Arr[n] - Arr[n+1] +1 = 증가값
ex) 4-2+1, 5-1+1
